Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -542,6 +542,26 @@ variable {R : Type u} [CommRing R] (E : EllipticCurve R)
 def j : R :=
   E.Δ'⁻¹ * E.c₄ ^ 3
 
+theorem j_eq_zero_iff' : E.j = 0 ↔ E.c₄ ^ 3 = 0 := by
```

Comment:
`lemma`? It's discussed somewhere that we might or might not reserve `theorem` for important theorems, but in any case the rest of this file follows the `lemma` convention.

Reply:
Done.